### ISSUE TITLE
Add tags for releases?

### ISSUE THREAD

**Issue by docwhat** [traefik/traefik-helm-chart#250]
Since #175 is a pain to do, could we add and maintain tags for each version?

Here's some shell commands for the current versions:

```sh
git tag 9.1.0 401c8cdf690cbbc765a935c7279566a13b79a082
git tag 9.0.0 6fa08574f26b1df4fa16b19100656e528c516a86
git tag 8.13.3 c174d5bb241bb567beb32022382b3eaf2a1e2c18
git tag 8.13.2 e71a733b3a4706eb6b2979e3c7ca6483c0dca958
git tag 8.13.1 2cda0fe18a7349e02fac7524c9640bf35a62c16c
git tag 8.13.0 98f4575d75b91c6e739ebc0e75c720c998f7dd38
git tag 8.12.0 4f43a379aa146e6166aa27bd459f331d132d679a
git tag 8.11.0 12368741468607a40b3919a24939d663d5eceef9
git tag 8.10.0 b348b3a84c4896d5d5a7d6d313b26d26c1051d75
git tag 8.9.2 37d8ea6c40158f51d3d087c82ecaf7c31b546673
git tag 8.9.1 8a64eda117418f9e2775b11a29a720e00cf199f1
git tag 8.9.0 75c6e11da1526126d4772d0f65f67ab5aac4a359
git tag 8.8.1 b7506c3e5b2354ef96a53920cb2e5ab984ecfe57
git tag 8.8.0 3ea03da56b335b6cf8c6bbeebd4d11dca6750968
git tag 8.7.2 c6e29bf6ee18e960bc2136129bdf93f2b8dc7d00
git tag 8.7.1 0e5bc75bbfcfd6784b36088a1350b9a24e8d1bce
git tag 8.7.0 0a6083f4e85e8e70d71bfd1b7a4b249bde89de90
git tag 8.6.1 9ca71391028cae10e2d83bdd3340662d1ab8de57
git tag 8.6.0 559204a921b5826ece52cdfbed146ec8665aa90c
git tag 8.5.0 4a1e6f9708bb3d3a187505e4e1dbf3e3cf800270
git tag 8.4.1 ecaf20236c93ce326588170dbb2548633ea7899e
git tag 8.4.0 289d2b2c93b9482d6483f19c06ac5619753c6b49
git tag 8.3.0 d6111482b416e13cf203ca608b36c6d5c0c3e226
git tag 8.2.1 7935e58067d9a35abd59d598173fe45c5affe344
git tag 8.2.0 732a34277e6270ed8831942f6b709750de8107ac
git tag 8.1.5 969b123bc8c2fe1e9f52e491f000b02a5b0cab61
git tag 8.1.4 15747eb0942477c8d309df3e0e36edee26d6d6dd
git tag 8.1.3 b3422bf90a6066d07766663ef36ef087959b0241
git tag 8.1.2 6cbeafd361ca6b28a6cff0147c1e97c90d31d183
git tag 8.1.1 f76731b637144eaf368156863549e94feb560501
git tag 8.1.0 07354a614ffbd31357fbb11b6be5bdb91d96f12e
git tag 8.0.4 87919cdbf6a8b81c4fa9d360a3bf5982e67b1f07
git tag 8.0.3 bc3771d11b7f3e58d2f4b6e974eed71efcad6f20
git tag 8.0.2 23722f3c670523624695dc336f57d488e9482b59
git tag 8.0.1 3b2133c6104031f89886d38f679eda05e03cbd8a
git tag 8.0.0 8895bc1d85ab882a38e73607c23cdc3a4efb1410
git tag 7.2.1 7602b382b536125307985c780e62dad53e827cc1
git tag 7.2.0 f6a6e976837eebd7b6da1b2dff14cd78a94b4ef1
git tag 7.1.0 6736a31626381a6bc769dc4f1e7e08d1d5a3e0a3
git tag 7.0.0 eaa29b04f70e545d744c2019037471649459de17
git tag 6.4.0 3d134f8d610d7ccc0061feb98843ec1e6bd12548
git tag 6.3.0 b60c4afd235b1cccc0f26fef25e4747fcbacb66a
git tag 6.2.0 e8f91fae14a05233dc1455e85e29b8343c027498
git tag 6.1.2 b4a7093b421cafb8e76ae8da61476d8d0eb91ef2
git tag 6.1.1 14a0fad3401960560fbe4c38e862cbce62b206d8
git tag 6.1.0 9705f69d2e308db3335821d24405a1062dc2ae32
git tag 6.0.2 ecc6215133de02abbca205904eb4c451bdab2ffa
git tag 6.0.1 edfc39cb4bb098fb21d9b144e98dce939ad6b408
git tag 6.0.0 8ccae234e254b2a1e561ad16b5423c0f3d991de8
git tag 5.6.0 9b4c8100adf94463a1b542fa21c87cca6abf1bcf
git tag 5.5.0 55798d155c3d73a89afeaa4d93764542ad14381c
git tag 5.4.0 0e107dad10c0b1908d5be54319f787fd6dd496a9
git tag 5.3.3 221a3387e581813abb4167179ce14feadb273275
git tag 5.3.2 92cd42237e067493e8579a29a147b3e95432ab4e
git tag 5.3.1 1a6c1785db88cbc8be1f193891984978d932e33a
git tag 5.3.0 39a2dadc0bbbf97099a28f4db1aa5ee3d62c492c
git tag 5.2.1 224bd15414c7ec08347e76d6f5aa892d3f19392d
git tag 5.2.0 f98b731274c2d8fc990571a160aa947c97e03632
git tag 5.1.0 93fce49ad79cf891b18164a36ec08fb2b09610d2
git tag 5.0.0 16604e67ba8206ab74af1959630c1bbabde18cf3
git tag 4.1.3 1faf247e2a5ce720f98b29d7de78c7003ea3d74f
git tag 4.1.2 45ae69f7d6276f67e5c2cdad4d5a75649b2019fd
git tag 4.1.1 632310e42e83d4f0cc09a12e78dd7fa0a1d43144
git tag 4.1.0 201af11adefde1e5bf8e754b9562e918ea023ccb
git tag 4.0.0 6b4a9a16bd02e4d83fc8f72ba259ac69d20b7799
git tag 3.5.0 43d63f418836d7aaac03cfca5b54f4287c8acf9c
git tag 3.4.0 3a7e78e7e007516454aa65b7e51c92fe3ca39ad3
git tag 3.3.3 3f13364367648af9ada3a7a6a5bb57b0fcdb9390
git tag 3.3.2 ae43a93829569246a1dd897fc7cb6d740794aaf3
git tag 3.3.1 76cc1a80eefe0a00d17c2c5c71340cead1bad22e
git tag 3.3.0 eb33a445a19ce15307bbc3a732c27c2c8bd668e8
git tag 3.2.1 ca4e233aed7c4600aac7fd3529982b6b976a2e42
git tag 3.2.0 3e3e9faa698e9d91eb3545ee5010e7f7227f5f0e
git tag 3.1.0 d7fc8d82551dd8c22a842948f32f4dc9233492e7
```

**Comment by SantoDE** [comment]
Just to make sure I understand you right: you want a git tag for every version of the helm chart? What is the benefit you're aiming for?

**Comment by SantoDE** [comment]
Alright. Thanks a lot. We will look into it 👍

**Comment by valvin1** [comment]
In my use case, i need tags to fix the chart helm I'm deploying using argocd. Right now each time you commit in master it desynchronizes argocd which requires an upgrade. Having tags would permit me to fix the helm chart version.
Right now I could do it but using commit hash directly which is not user friendly.

**Comment by ldez** [comment]
Fixed by ba98e364cf82d3c0d179b321aa668d53dc10e030
